lib/embedded_tweet_view.dart

```diff
@@ -46,6 +49,7 @@ class EmbeddedTweetView extends StatelessWidget {
     this.onTapImage,
     this.createdDateDisplayFormat,
     required this.videoHighQuality,
+    this.showRepliesCount = false,
```

Comment:
We just add it to embedded tweet view?

Reply:
Well, from the examples, I only see the public metrics only displayed on the Embedded tweets. And when I tried to look at the Tweet and Compact views, I can't get clearly where to add this 🙈.

I'm a noob as far as frontend is concerned. Been a backend developer my whole life. Now I'm just converting to full-stack

<img width="1072" alt="Screenshot 2022-04-08 at 23 54 26" src="https://user-images.githubusercontent.com/22393543/162537522-80537ba4-c720-485d-a2e7-18ef53dcb6ad.png">
.

Reply:
ok, then let's leave it like this.